packages/recoil-sync/package-for-release.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "recoil-sync",
-  "version": "0.1.0",
+  "version": "0.1.1-alpha.1",
```

Comment:
There hasn't been a `0.1.0` release yet, so I think you could still use `0.1.0-alpha.1`

Reply:
Ah you are right. I'll change it to 0.1.0. When we officially release it we can call it 0.1.0, but I won't publish a 0.1.0-alpha.1 unless it become necessary